### PR TITLE
Cleanup code and docs

### PR DESCRIPTION
**Pull request by adrw**
In advance of 1.0.0, I'm doing a pass through the repo to cleanup code and docs and remove unused APIs.

### CODE REVIEW

File: samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Square, Inc.
+ * Copyright (C) 2022 Block, Inc.
```
Comment (JakeWharton) [inline samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt]: I was always told to never change these dates.

As to changing the copyright holder name, I would recommend consulting our open source legal people to ensure this is valid to do.

**Review by JakeWharton — APPROVED** [review]

File: samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt (hunk 1 above)
Comment (swankjesse) [reply samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt]: More specifically:
* Don’t change these dates ’cause it’s not useful.
* Do use '2022 Block, Inc' in new code

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt [hunk 2]
```diff
@@ -94,6 +94,7 @@ class ZiplineLoader internal constructor(
     maxSizeInBytes: Long,
   ): ZiplineLoader {
     fileSystem.createDirectories(directory, mustCreate = false)
+    // TODO rename DB name to zipline.db once schema is stable
```
Comment (swankjesse) [inline zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt]: I believe the schema is stable now

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt [hunk 3]
```diff
@@ -125,6 +126,8 @@ class ZiplineLoader internal constructor(
   }
 
   private var concurrentDownloadsSemaphore = Semaphore(3)
+
+  /** Callers can modify this as desired to change the default network download concurrency level. */
   var concurrentDownloads = 3
```
Comment (swankjesse) [inline zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt:130]: Now that we know we have 33 files in payment-history I think we should set this limit to 16.

**Review by swankjesse — APPROVED** [review]

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt (hunk 3 above)
Comment (swankjesse) [reply zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt:130]: (we can do this in the mobile-multiplatform repo)

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt (hunk 2 above)
Comment (JakeWharton) [reply zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt]: Tracked by #627!

File: samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt (hunk 1 above)
Comment (adrw) [reply samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt]: Reverted everything and will stick with `current year Block, Inc` on new files

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt (hunk 3 above)
Comment (adrw) [reply zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt:130]: Sounds good, I'll leave the default as is in Zipline and update in MMP

**Pull request merged**